widgets/src/components/TaskDetails/Overview/OverviewDetails.js

```diff
@@ -27,7 +25,7 @@ const displayedDetails = [
   {
     label: 'task.fields.status',
     value: 'status',
-    render: value => <BadgeChip label={value} />,
```

Comment:
We still should be using the React components internally.

Reply:
Shouldn't we use the MFEs to actually separate stuff?